dozer/cogs/moderation.py

```diff
@@ -36,22 +36,31 @@ def __init__(self, bot):
 
     """=== Helper functions ==="""
 
-    async def mod_log(self, actor: discord.Member, action: str, target: Union[discord.User, discord.Member], reason, orig_channel=None,
+    async def mod_log(self, actor: discord.Member, action: str, target: Union[discord.User, discord.Member, None], reason, orig_channel=None,
                       embed_color=discord.Color.red(), global_modlog=True):
         """Generates a modlog embed"""
+
+        if target is None:
+            title = "Custom Modlog"
+        else:
+            title = f"User {action}!"
+
         modlog_embed = discord.Embed(
             color=embed_color,
-            title=f"User {action}!"
+            title=title
 
         )
-        modlog_embed.add_field(name=f"{action.capitalize()} user", value=f"{target.mention} ({target} | {target.id})", inline=False)
+        if target is not None:
+            modlog_embed.add_field(name=f"{action.capitalize()} user", value=f"{target.mention} ({target} | {target.id})", inline=False)
         modlog_embed.add_field(name="Requested by", value=f"{actor.mention} ({actor} | {actor.id})", inline=False)
         modlog_embed.add_field(name="Reason", value=reason or "No reason specified", inline=False)
-        modlog_embed.add_field(name="Timestamp", value=str(datetime.datetime.now()), inline=False)
+        modlog_embed.timestamp = datetime.datetime.utcnow()
         try:
             await target.send(embed=modlog_embed)
         except discord.Forbidden:
             await orig_channel.send("Failed to DM modlog to user")
+        except AttributeError:
+            pass
```

Comment:
This could mask actual bugs in the code if something other than a User/Member is provided. This should be handled with a check if `target is None` before sending the DM.

Reply:
Ok